.github/workflows/callable_test-automation.yaml

```diff
@@ -1,7 +1,7 @@
 name: indy-test-automation
 on:
-  # push:
-  #   branches: [sovrin]
+  push:
+    branches: [main]
```

Comment:
```
  push:
    branches: [main]
```
This should actually be removed.  The workflow should only be triggered through a workflow dispatch or workflow call.